testrun.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+#chmod 755 ./testrun.sh
+
+function EPHEMERAL_PORT() {
+	LOW_BOUND=49152
+	RANGE=16384
+	while true; do
+		CANDIDATE=$(($LOW_BOUND + ($RANDOM % $RANGE)))
+		(echo "" > /dev/tcp/127.0.0.1/${CANDIDATE}) > /dev/null 2>&1
+		if [ $? -ne 0 ]; then
+			echo $CANDIDATE
+			break
+		fi
+	done
+}
+
+function CLEAN_AFTER_TEST() {
```

Comment:
Nit: you can use Makefile for it.

Reply:
How Makefile can help me?

[testrun.sh](https://github.com/DimaKoz/practicummetrics/pull/10/files/7e0379b389a7998df613f03d488aeab0e1d72f2b#diff-65bead77c2a7238416ac582fd63c6295419d612b0a00ceb2290566664de43e20) file is not particularly relevant to the project, I just got tired of constantly typing some commands(like `metricstest-darwin-amd64 -test.v -test.run=^TestIteration7$ -server-port="$RANDOM_PORT" -agent-binary-path=./ag -binary-path=./ser -source-path=.` in the terminal to run Yandex autotests locally, this file makes my work easier in the form in which it is now. I can remove it from the repository completely. What do you think?

Reply:
Some explanations added to the file.
done

Reply:
This is a useful file. But what if you want to run a concrete test? 
What if you want to clean the dirs? 
In my point of view, you can use commands for it. 
Anyway, please look at it https://earthly.dev/blog/golang-makefile/